tap_gitlab/__init__.py

```diff
@@ -932,12 +972,7 @@ def main_impl():
     args.config["private_token"] = args.config["private_token"].strip()
 
     CONFIG.update(args.config)
-    CONFIG['ultimate_license'] = truthy(CONFIG['ultimate_license'])
-    CONFIG['fetch_merge_request_commits'] = truthy(CONFIG['fetch_merge_request_commits'])
-    CONFIG['fetch_pipelines_extended'] = truthy(CONFIG['fetch_pipelines_extended'])
-    CONFIG['fetch_retried_jobs'] = truthy(CONFIG['fetch_retried_jobs'])
-    CONFIG['fetch_group_variables'] = truthy(CONFIG['fetch_group_variables'])
-    CONFIG['fetch_project_variables'] = truthy(CONFIG['fetch_project_variables'])
```

Comment:
Why remove these?

Reply:
It's a mistake on my side :/ I had to fight with my automatic formatter. I will redo my commit 👍 😅

Reply:
Should be good now 🤔